core/src/components/icon/test/basic/index.html

```diff
@@ -143,7 +152,7 @@ <h3>mode: md</h3>
       dynamicHomeIcon.name = 'home';
       dynamicHomeIcon.isActive = false;
 
-      icons = ['home', 'star', 'ios-alert', 'ios-alert-outline', 'md-alert', 'logo-apple'];
+      icons = ['home', 'star', 'alert-circle', 'alert-circle-outline', 'alert-circle-sharp', 'logo-apple'];
```

Comment:
I updated this to use the latest Ionicon names by comparing the older names with the v3 Ionicons to see what it should be now: https://ionicframework.com/docs/v3/ionicons